docs/03_tutorials/20_tic-tac-toe-game-smart-contract-single-node.md

```diff
@@ -644,7 +646,7 @@ cleos set contract host ./ tictactoe.wasm tictactoe.abi -p host@active
 Now that the smart contract has been successfully deployed, [push smart contract actions](https://developers.eos.io/manuals/eos/latest/cleos/command-reference/push/push-action) to the blockchain to play the game.  
 
 
-### Push 'create' to the single node testnet to create a game
+### Push 'create' to the Smart Contract to create a game
 A game requires a host and a challenger. Use the accounts created earlier in the “Create the necessary accounts and key pairs” sectio of the tutorial for these. These accounts use arbitrary names. In this example assume the host has the account name of ‘mcazyfujecke’ and the challenger has the account name of ‘vswlkiegwdsk’.
 
 The create action takes two parameters, the "challenger" and the "host". The required payload in json format is:
```

Comment:
The \`create\` action
Sign the push action with \`host@active\`

Reply:
see slack for comment about using lots of ``

Reply:
Actions should be emphasized with backticks except in titles and headings.